docker/docker-compose.mysql.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  web:
+    image: tsp:latest
+    build: .
+    volumes:
+      - .:/app
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - ~/.Xauthority:/root/.Xauthority
```

Comment:
I'm not a docker or ruby expert, but mounting `/tmp/.X11-unix` and `Xauthority` into the container looks strange for a webapp.

Is there a special reason why you do this?

Reply:
Yes, there is a reason: running the integration tests using a browser. In this case, you need to allow `capybara-webkit` to connect to your X server. As said before, this configuration is oriented for development only :)

Of course, when deploying the application, you shouldn't do that.

Reply:
I will try to make more clear that this docker-compose thingie is just for development and I will provide a production oriented docker manifest.

Reply:
Thanks for the explanation.

As an idea - renaming this file to something like `docker-compose.development.mysql.yml` would be a good start ;-)